frontend/pictonode-web/src/app/Widgets.vue

```diff
@@ -67,23 +67,51 @@ export default defineComponent({
       items: categorize(),
     };
   },
+
   methods: {
     addNode(item: Item) {
       this.$emit("input", [...this.pendingTemplates, item.templateName]);
     },
+
+    getTooltip(item: string): string {
```

Comment:
I'd rather attach this as metadata associated with the template, like we do for the names and categories.

Reply:
Then again, that's splitting hairs, I'm fine with this.